src/main.rs

```diff
@@ -52,6 +52,7 @@ fn main() -> Result<(), Error> {
 
     let top_ns = std::env::var("KUBERNETES_NAMESPACE").unwrap_or_else(|_| DEFAULT_NAMESPACE.into());
     let top_cfg = kubeconfig().expect("Load default kubeconfig");
+    info!("apiserver:{}", top_cfg.base_path);
```

Comment:
is this a debug message?

Reply:
I intentionally made it as info as we want to know the apiserver all the time.

Reply:
it's ok